.github/ISSUE_TEMPLATE/questions_and_others.md

```diff
@@ -0,0 +1,10 @@
+---
+name: "❓ Questions and Others"
+about: Questions regarding Klaytn DEX or other issues not related to bug nor feature request
```

Comment:
as we have repo for backend and frontend, best to reflect it here as well thanks

Reply:
sure